pkg/option/flags.go

```diff
@@ -205,6 +205,7 @@ func AddFlags(flags *pflag.FlagSet) {
 	flags.String(KeyK8sKubeConfigPath, "", "Absolute path of the kubernetes kubeconfig file")
 	flags.Bool(KeyEnableProcessAncestors, true, "Include ancestors in process exec events")
 	flags.String(KeyMetricsServer, "", "Metrics server address (e.g. ':2112'). Disabled by default")
+	flags.String(KeyMetricsLabelFilter, "", "Metrics Label Filter Comma-separated list of enabled metric labels. By default all labels are enabled.")
```

Comment:
```suggestion
	flags.String(KeyMetricsLabelFilter, "", "Comma-separated list of enabled metric labels. By default all labels are enabled.")
```
No need to repeat the flag name in the help text.